### DISCUSSION TITLE
web UI Preview

### DISCUSSION THREAD

**Discussion by augugusto** [mtotschnig/MyExpenses#1260] [General]
Sorry i'm opening a discussion here, but i didn't want to create an issue for this. 

I have two questions:

1.  Where can i preview the web UI feature? the idea appeals to me to compare my own records to my bank, but i'd like to know if the features works for me beofre buying
2. If i compile from source. Do i still need a licence? i suspect i probably can make it work by editing the code, but i'll probably not do that. if building from source give it for free, great, if not, i'll cosider buying a licence

**Comment by augugusto** [comment]
actually, if someone can point me on how to change the code so that it doesn't require the licence, i'd be grateful too, but of this comment is not ok with the owner or the community, i'll remove it. my main questions is are the ones on the main post

**Comment by mtotschnig** [comment]
Every feature can be tested for 2 months. The Web User Interface is activated from the Settings, section Features.

**Reply by augugusto** [reply]
i cannot preview it. i've got the fdroid version of the app. it just takes me to paypal

**Reply by mtotschnig** [reply]
If you click on Start on the Web User Interface button, it opens the Premium feature dialogue, which has a Try button at the bottom. Maybe, on phones with narrow screen, you have to scroll in order to make it possible

**Comment by augugusto** [comment]
besides the fact that i cant try the web UI, i also cannot export my data. why?

**Reply by mtotschnig** [reply]
What type of export did you try? What problem did you encounter?

**Comment by augugusto** [comment]
I apologize for my vague messages. I was able to try the web ui. The "try" button is greyed out and it looks disabled.

I think I will also be able to export. I haven't tried it yet. I look for the option to export everywhere in settings. Didn't think of looking in the quick actions menu

**Comment by mtotschnig** [comment]
Thanks for the feedback about the Try button. I will remove the grey color in next version.
